spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,103 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
```

Comment:
I don't think "must" will actually make the language stronger, because the server can just pretend that the error can't be associated with a particular part of the result. No matter what we do, some errors won't have a `path`, so clients need to do something sane in that case.